eng/tools/rush-runner.js

```diff
@@ -167,7 +167,8 @@ if (serviceDirs.length === 0) {
   spawnNode(baseDir, "common/scripts/install-run-rush.js", action, ...rushParams);
 } else {
   let params = [];
-  switch (action.toLowerCase().split(":")[0]) {
+  const actionComponents = action.toLowerCase().split(":");
+  switch (actionComponents[0]) {
     // case 'build':
     //   params = flatMap(packageNames, (p) => [`--to`, p, `--from`, p]);
     //   spawnNode(baseDir, 'common/scripts/install-run-rush.js', action, ...params, ...rushParams);
```

Comment:
perhaps this should be deleted now

Reply:
@praveenkuttappan -- is this fragment left over from your work on dependency calculation? Is it safe to say now that we can remove it?

Reply:
This can be removed. I think this is left over from long back when issue in rush was found and disabled selective build.